src/Topics/StandardModernPrecision/Bids1D.hs

```diff
@@ -128,7 +129,8 @@ b1D3C = do
                  , suitLength T.Diamonds 4 >> minSuitLength T.Clubs 5 ]
     mapM_ (`maxSuitLength` 3) T.majorSuits
     makeAlertableCall (T.Bid 3 T.Clubs)
-        "5-4 or 4-5 in the minors, less than invitational strength"
+        ("5" .+ NDash .+ "4 or 4" .+ NDash .+ "5 in the minors, " .+
```

Comment:
Should this use hyphens or dashes? I'm honestly unsure. :thinking:

Reply:
https://en.wikipedia.org/wiki/Dash#Relationships_and_connections implies that an en-dash is the right choice.